src/analytics/useAnalyticsAttributes.ts

```diff
@@ -14,7 +14,6 @@ export const useAnalyticsAttributes = () => {
 
   useEffect(() => {
     if (typeof newrelic !== "object") {
-      console.log("Setting userId: ", userId);
       return;
```

Comment:
what kind of failure does this log help debug? I'm wondering if we can report something to bugsnag or add a unit test instead of log to the dev console.

Reply:
This log only gets displayed when new relic is not in scope. I believe new relic is only in scope when we have a production build of spruce.
So for local development it allows us to confirm our analytics calls are being properly triggered.

Reply:
So if we perform some action when testing something locally we get immediate confirmation what ever we did is properly calling new relic. I've personally found it useful in the past

Reply:
I added the logging since it helpful for now and opened [a ticket](https://jira.mongodb.org/browse/EVG-13518) that will hopefully make us confident in removing the log line in the future